.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,55 @@
+name: Contract Deployments Integration Test
+
+on: pull_request
+
+jobs:
+    test:
+        runs-on: ${{ matrix.os }}
+        strategy:
+            matrix:
+                os: [ubuntu-22.04]
+                arch: [amd64]
+        steps:
+            - name: Setup Node
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 18
+
+            - name: Checkout code
+              uses: actions/checkout@v3
+
+            - name: Install
+              run: npm ci
+
+            - name: Spin up Hardhat Network
+              run: npx hardhat node &
+
+            - name: Prepare local.json
+              run: |
+                  echo '{
+                    "chains": {
+                      "ethereum": {
+                        "name": "Hardhat",
+                        "id": "hardhat",
+                        "chainId": 31337,
+                        "rpc": "http://127.0.0.1:8545",
+                        "tokenSymbol": "ETH",
+                        "contracts": {}
+                      }
+                    }
+                  }' > axelar-chains-config/info/local.json
+
+            - name: Display local.json
+              run: cat axelar-chains-config/info/local.json
+
+            - name: Deploy constAddressDeployer
+              run: node evm/deploy-contract.js -e local -a ../node_modules/@axelar-network/axelar-gmp-sdk-solidity/artifacts/contracts/deploy/ -c ConstAddressDeployer -n ethereum -m create -p 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 --yes
```

Comment:
test private key should be added to an `.env` to avoid this duplication. Similarly, `.env` can contain `ENV`, `CHAIN`